Samples/london-no2.py

```diff
@@ -1,19 +1,40 @@
+'''
+This recipe will get you started with using Digital Connector, 
+it uses the minimal objects required to built a useful recipe.
+
+The recipe aims at giving the latest value of No2 40 ug/m3 annual mean 
+for every LocalAuthority
+'''
+
 from os import path, pardir
 import sys
 sys.path.append(path.join(path.dirname(path.realpath(__file__)), pardir))
 
+# Declaring the path of Digital Connector
+# Declaring the path of the generated recipe, that would save the recipe in json format 
+# before sending it to DC.
+# Declaring the location of the outut file
+# Note: all three declaration should be relative to user's home directory
 tombolo_path = 'Desktop/TomboloDigitalConnector'
 recipe_output_location = 'Desktop/london-no2.json'
 model_output = 'Desktop/london-no2.geojson'
 
+# importing the required classes for building the recipe
 from recipe import Recipe, Field, Datasource, AttributeMatcher, Subject, Match_Rule, LatestValueField, Dataset
 
+# Creating Subject and Datasources to tell DC which importer needs to be called 
+# inorder to dowload datasets for more info please refer to documentation
```

Comment:
typo in order
typo download
... to run the recipe. For more info ...

Reply:
done